admin/commands/storage/read_seals.go

```diff
@@ -71,7 +71,7 @@ func (r *ReadSealsCommand) Handler(ctx context.Context, req *admin.CommandReques
 		if err != nil {
 			return nil, fmt.Errorf("failed to retrieve index for block %#v: %w", blockID, err)
 		}
-		lastSeal, err := r.seals.ByBlockID(blockID)
+		lastSeal, err := r.seals.HighestInFork(blockID)
```

Comment:
This PR took the chance to rename this function in order to make it more clear as per the review suggestion